app/main/views/briefs.py

```diff
@@ -546,3 +546,37 @@ def list_brief_services(brief_id):
 
     services = services.filter(Service.supplier_id == supplier.supplier_id)
     return list_result_response("services", services), 200
+
+
+@main.route('/briefs/<int:brief_id>/move-unsuccessful-brief-to-closed', methods=['POST'])
+def move_unsuccessful_brief_to_closed(brief_id):
+    """
+    Status changes:
+        Brief: unsuccessful -> closed
+    """
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if brief.status != 'unsuccessful':
+        abort(400, "Can only move an unsuccessful brief to close")
+
+    # Set status back to pending-awarded
```

Comment:
Is this true? It doesn't match with the method name and doc comment.

Otherwise, looks good.

Reply:
I'm not sure I understand what you mean? How does it not match the method name and doc comment, I'm not sure what I'm missing.

Reply:
Are we setting the status back to `pending-awarded` or `closed`? Or to those two mean the same thing?

Reply:
Just looked it up. `pending-awarded` is not a Brief status - it's a Brief Response status.

Reply:
Fortunately, we don't have to worry about brief responses with this as an unsuccessful brief will not have any

Reply:
Sorry - I should have been clearer. 

```suggestion
    # Set status back to closed
```